### DISCUSSION TITLE
Support for Multi-Select Picklist

### DISCUSSION THREAD

**Discussion by clurdish** [Innofactor/XrmToolBox.BulkDataUpdater#215] [Ideas]
First off - thank you for building Bulk Data Updater. It's a great tool that's very convenient for admin tasks.

I noticed while trying to use the tool to back fill default values for a multi-select choice column that the column is being identified as a "Virtual" type. I'm assuming this is also just the default for "unknown" types. 

It would be great to be able to use this tool for multi-select choice columns. Maybe there could be an option to add the value to a selection if it's missing as well as another option to set the fixed set of selected values.

Let me know your thoughts!

**Comment by rappen** [comment]
Duplicated with #213 
This issue is fixed; the release will be coming "soon".
I had a hiccup there, not sure when and why though.... 🙄
